nova_api/exceptions/__init__.py

```diff
@@ -80,3 +80,9 @@ class NoRowsAffectedException(NovaAPIException):
     message: str = field(default="No rows were affected by the "
                                  "desired operation", init=False)
     error_code: int = field(default=304, init=False)
+
+
+@dataclass
+class InvalidAttributeException(NovaAPIException):
+    status_code: int = field(default=400)
+    message: str = field(default='Invalid attribute value')
```

Comment:
```suggestion
    status_code: int = field(init=False, default=400)
    message: str = field(init=False, default='Invalid attribute value')
```
Quando tiramos o status_code e o message do init, o `debug` se torna o primeiro argumento, dessa forma, `raise InvalidAttributeException("This is debug")` define corretamente o debug mais facilmente.

Reply:
Bacana, não sabia dessa